static/css/app.css

```diff
@@ -0,0 +1,2773 @@
+@font-face {
```

Comment:
If this is a built file, we shouldn't check it into git, right?

Reply:
@groovecoder Yeah – It's left over while we were having build issues. Let me update this PR

Reply:
It has been removed. 😅